### PR TITLE
Feature/sn 643 add bulk delete

### PR DESCRIPTION
**Pull request by AlexanderMakarov124**
### Button and checkboxes
image

### Dialog
image

### Warning when no records were selected
image

### CODE REVIEW

**Comment by vasiliy-chefonov** [comment]
Let's just disable the button in case no records were selected.

File: src/Saritasa.NetForge.Blazor/Pages/EntityDetails.razor.cs [hunk 1]
```diff
@@ -91,6 +93,36 @@ private async void ShowDeleteEntityConfirmationAsync(object source)
         }
     }
 
+    private async Task ShowBulkDeleteEntitiesConfirmationAsync()
+    {
+        if (ViewModel.SelectedEntities is null || !ViewModel.SelectedEntities.Any())
```
Comment (vasiliy-chefonov) [inline src/Saritasa.NetForge.Blazor/Pages/EntityDetails.razor.cs:98]: Can we initialize with the empty collection?

File: src/Saritasa.NetForge.Blazor/Pages/EntityDetails.razor.cs [hunk 2]
```diff
@@ -91,6 +93,36 @@ private async void ShowDeleteEntityConfirmationAsync(object source)
...
+
+        var parameters = new DialogParameters
+        {
+            { nameof(ConfirmationDialog.ContentText), "Are you sure you want to delete these records?" },
+            { nameof(ConfirmationDialog.ButtonText), "Yes" },
+            { nameof(ConfirmationDialog.Color), Color.Error }
+        };
+
+        var result = await (await DialogService.ShowAsync<ConfirmationDialog>("Bulk Delete", parameters)).Result;
+        if (!result.Canceled)
```
Comment (vasiliy-chefonov) [inline src/Saritasa.NetForge.Blazor/Pages/EntityDetails.razor.cs:112]: Can we invert the if statement?

**Review by vasiliy-chefonov — APPROVED** [review]

**Comment by vasiliy-chefonov** [comment]
Please, hide the Bulk delete button on the keyless entities.

**Review by mchernikov80 — APPROVED** [review]

**Pull request merged**